package.json

```diff
@@ -47,6 +47,7 @@
     "@babel/runtime": "^7.14.6",
     "babel-runtime": "~6.25.0",
     "color": "^3.1.3",
+    "html-minifier-terser": "*",
```

Comment:
Could you make it optional dep https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies ?

Reply:
I could do that, but I am wondering if that's a good idea? I guess everybody would like to minify their emails as this impacts the delivery ratio?

Reply:
Or another option is not to give any minify option at all. Users will figure out themselves what library to use.

Reply:
E.g. we use different minify library in our project.

Reply:
What do you use? I added this as an option only because I wanted to give users ability to turn it off in case they find minifier is causing some issues... Should I turn it on by default?

Reply:
Turning it off is good change. I just think it is better not to add any dep for minify.

Reply:
I hear you, but why?

Reply:
That's additional dep and potential source of problems.

Reply:
What kind of problems. Its part of `terser` repo - well maintained.

Reply:
If it becomes a problem, it can always be replaced with something else...

Reply:
Still it is additional things to install and it is like death by a thousand cuts.

Reply:
Depends if its the main flow or not. And for email, I would assume it should default to get minified html...?

Reply:
OK. Let's wait a little bit then. In month or two your PR will be good to go.